services/brig/test/integration/API/BrowseTeam.hs

```diff
@@ -0,0 +1,83 @@
+module API.BrowseTeam where
```

Comment:
```suggestion
module API.BrowseTeam (tests) where
```

I think this will show you that `assertBrowseTeamCannotFind` is not called anywhere, and remind you that you should add another test case.